libs/community/tests/unit_tests/vectorstores/test_indexing_docs.py

```diff
@@ -81,5 +81,6 @@ def check_compatibility(vector_store: VectorStore) -> bool:
         "ZepVectorStore",
         "Zilliz",
         "Lantern",
+        "OpenSearchVectorSearch",
```

Comment:
can we add OpenSearchVectorStore to the list in docs/docs/modules/data_conections/indexing.ipynb

Reply:
I'll do that, and I have a question.
Should I change the name OpenSearchVectorSearch to OpenSearchVectorStore?

Reply:
yep, sorry typo on my end. Let's keep as OpenSearchVectorSearch